db/schema.rb

```diff
@@ -552,6 +560,16 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
   end
 
+  create_table "versions", force: :cascade do |t|
```

Comment:
What is this table for?

Reply:
Hi @compwron It was just there when I pulled the current main branch. Is it not in the current main branch?